lib/MetaCPAN/Server/QuerySanitizer.pm

```diff
@@ -50,10 +50,13 @@ sub _scan_hash_tree {
     my $ref = ref($struct);
     if ( $ref eq 'HASH' ) {
         while ( my ( $k, $v ) = each %$struct ) {
-            if ( $k eq $key ) {
-                MetaCPAN::Server::QuerySanitizer::Error->throw(
-                    message => qq[Parameter "$key" not allowed], );
-            }
+            # Mickey: disabling this check for 'script' key since
+            #         for ES 1.7 I need to use it in the
+            #         function_score syntax
```

Comment:
restored in 97c61e0

note that in ES2 we need to add the following configuration to `elasticsearch.ym`:
```
script.inline: on
```